homeassistant/config_entries.py

```diff
@@ -480,7 +480,7 @@ async def async_unload(
 
             self._async_process_on_unload()
 
-            return result
+            return result  # type: ignore[no-any-return]
```

Comment:
Opened a new issue in mypy. Could you include the link here? https://github.com/python/mypy/issues/11839